docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -22,6 +22,9 @@ To enable OAuth 2.0 authentication for Trino, configuration changes are made on
 the Trino coordinator. No changes are required to the worker configuration;
 only the communication from the clients to the coordinator is authenticated.
 
+When configuring an OAuth 2.0 authorization server like an OpenID-connect provider,
+The callback/redirect URL should be `https://<trino-coordinator>:<https-port>/oauth2/callback`.
```

Comment:
turn around and make active voice

Set the callback/redirect to .. , when configuring...

Reply:
Thanks! fixed in https://github.com/trinodb/trino/pull/8198/commits/d2f1b4250279b2417fb0ecd7a522ce5091d207f9